### PR TITLE
Fix RFQ Button

### PR DESCRIPTION
**Pull request by zhouyuhe**
Change the RFQ position:  RFQ button is now aligned with the price unavailable boxes

### CODE REVIEW

**Comment by Sharakai** [comment]
Looks like you have 2 merge commits in the commit history of this branch, which is a little odd. I'd recommend `rebase`ing instead of merging.

Makes sure this is squashed and the commit message corrected when this is pushed into the `master` branch

File: src/client/package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
   },
   "scripts": {
     "precommit": "lint-staged",
-    "start": "cross-env REACT_APP_BUILD_VERSION=$(git describe) react-scripts start",
+    "start": "react-scripts start",
```
Comment (Sharakai) [inline src/client/package.json]: Q: How come this was changed? Looks unrelated to the issue at hand.

File: src/client/src/apps/MainRoute/widgets/spotTile/components/notifications/TileBooking.tsx [hunk 2]
```diff
@@ -30,6 +30,8 @@ const BookingPill = styled.div<{ disabled: boolean; altStyle: boolean; color: st
     !disabled &&
     `
   cursor: pointer;
+  position: relative;
+  top:-8px;
```
Comment (Sharakai) [inline src/client/src/apps/MainRoute/widgets/spotTile/components/notifications/TileBooking.tsx]: You're using absolute units to position this, but padding is in relative units.
Would reducing the `padding` fix this, rather than having to use a negative placement?

Comment (Sharakai) [reply src/client/src/apps/MainRoute/widgets/spotTile/components/notifications/TileBooking.tsx]: Make sure that this looks okay at multiple zoom levels.
I recommend 75%, 100%, and 150%, and 200% at a minimum.

File: src/client/package.json (hunk 1 above)
Comment (rikoe) [reply src/client/package.json]: We changed it to get `npm start` working on @zhouyuhe 's machine - I added that bit, but it only seems to work on Mac/Unix 😞

Comment (zhouyuhe) [reply src/client/package.json]: Because with `REACT_APP_BUILD_VERSION=$(git describe)` it shows the error, and I cannot run the app

File: src/client/src/apps/MainRoute/widgets/spotTile/components/notifications/TileBooking.tsx (hunk 2 above)
Comment (zhouyuhe) [reply src/client/src/apps/MainRoute/widgets/spotTile/components/notifications/TileBooking.tsx]: Good spot, will change it

**Comment by rikoe** [comment]
Closing as we are revisiting the design.
